lib/factory.rb

```diff
@@ -14,3 +14,125 @@
 # - to_a
 # - values_at
 # - ==, eql?
+class Factory
+    include Enumerable
+    def self.new(*key_args, keyword_init: false, &block)
```

Comment:
1) please use intentations between methods -  it's almost non-readable
2) What does that lines in Gemfile mean? It causes error.
```
MyClass = Stuct.new do
  def my_method
    'Method call'
  end
```
3. Please add `rubocop` to Gemfile
4. Why do we need `Enumerable` module here?